src/transactions/SponsorshipUtils.cpp

```diff
@@ -753,6 +753,13 @@ removeEntryWithPossibleSponsorship(AbstractLedgerTxn& ltx,
 
         if (acc.current().data.account().accountID == *le.ext.v1().sponsoringID)
         {
+            // the "owner" of a subentry cannot also be the sponsor of that
```

Comment:
What does this have to do with this PR?

Reply:
It doesn't. I wanted to add this comment and thought I'd squeeze it into this PR instead of creating a new one. I can open a separate PR instead.